community/bolt/src/docs/dev/examples.asciidoc

```diff
@@ -438,8 +438,8 @@ Server: SUCCESS {
 ----
 === Accessing notifications
 When Neo4j executes a statement it may include notifications for the user.
-These notifications can be warnings about problematic statements or other valuable information for a client. 
-Unlike failures or errors, notifications do not affect the execution of a statement.
+These notifications can be warnings about problematic statements or other valuable information for a client.
+Notifications are only included when using `EXPLAIN`.
```

Comment:
Is this true as a hard fact, or is it just de-facto currently? I thought Cypher could trigger notifications whenever it liked - but I'm realizing as I type this that I just assumed that.